models/articles.model.js

```diff
@@ -64,9 +64,24 @@ function insertCommentByArticleId(username, body, article_id) {
     });
 }
 
+function updateVotesByArticleId(inc_votes, article_id) {
+  return db
+    .query(
+      `UPDATE articles
+      SET votes = votes + $1
+      WHERE article_id = $2
+      RETURNING *;`,
```

Comment:
lovely! We may need to add some error handling here once we add more error tests